template.tpl

```diff
@@ -33,14 +33,14 @@ ___TEMPLATE_PARAMETERS___
   {
     "type": "TEXT",
     "name": "api_key",
-    "displayName": "PushNinja Key",
+    "displayName": "SiteRecording Key",
     "simpleValueType": true,
     "valueValidators": [
       {
         "type": "NON_EMPTY"
       }
     ],
-    "help": "Please copy the PushNinja Key from the share script sidebar in the application."
+    "help": "Please copy the SiteRecording Key from Website -\u003e Share/Copy script sidebar -\u003e Connections -\u003e Google Tag Manager at https://infinity.500apps.com/siterecording"
```

Comment:
What is this special character? \u003?

Reply:
sir, its encoded as we export the file from "->" to \u003e